lints/lint_qcstatem_etsi_type_as_statem_test.go

```diff
@@ -0,0 +1,39 @@
+package lints
+
+/*
+ * ZLint Copyright 2018 Regents of the University of Michigan
+ *                 2018 cryptosource GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import (
+	"testing"
+)
+
+func TestEtsiTypeAsQcStmt(t *testing.T) {
```

Comment:
It seems like pretty consistently `Statem` is used as the abbreviation for `Statement`, but here it has `Stmt`?

`Statem` presents its own problems (e.g. concating `Statem` and `etsi` looks like a typo for `Statements` with the `n` missing), but it seems to win out over `Stmt` in this PR)

Reply:
Yes, this should be changed to Statem